cmd/releasego/templates/announcement.template.md

```diff
@@ -0,0 +1,17 @@
+---
+post_title: {{.Title}}
+author1: {{.Author}}
+post_slug: {{.Slug}}
+categories: {{.CategoriesString}}
+tags: {{.TagsString}}
+featured_image:
+summary: The Microsoft builds of the Go security patches released today, are now available for download.
+---
+
+The Microsoft builds of the Go security patches released today, are now [available for download](https://github.com/microsoft/go#binary-distribution). For more information about this release and the changes included, see the table below:
```

Comment:
The way I read this now, it says that the Go security patches were released today (which might not be the case if it takes us a day or more to release a new Microsoft Go), and the first `,` is a comma splice. (I can see how another interpretation might make sense, but it isn't how I'd normally read it. 😄)

Also, it always says "security patches", but that might not be the case.

I think simplifying it helps solve both of those things:

```suggestion
A new set of Microsoft Go builds is now [available for download](https://github.com/microsoft/go#binary-distribution).
{{- if .SecurityRelease }}This release includes security fixes from an upstream Go release.{{ end -}}
For more information about this release and the changes included, see the table below:
```

I haven't tested out this suggestion, and it means the input data needs a new field! I think simply removing the mention of security releases for now is ok, and this can be addressed in a followup PR:

```suggestion
A new set of Microsoft Go builds is now [available for download](https://github.com/microsoft/go#binary-distribution). For more information about this release and the changes included, see the table below:
```

Reply:
fixed